### ISSUE TITLE
crash in libpangoft2-1.0.so while spectating

### ISSUE THREAD

**Issue by Turmfalke2** [ValveSoftware/Dota-2#1967]
hi,

I was spectating a game, when they unpaused the screen froze and a bit later the game crashed.

```
[40627.866928] GlobPool/2[231585]: segfault at 0 ip 00007fbeeafad108 sp 00007fbef33f3938 error 4 in libpangoft2-1.0.so[7fbeeaf17000+131000]
[40627.866937] Code: 00 00 48 8b 57 18 48 3b 57 10 72 1e 48 8b 17 31 c0 48 85 d2 74 0c 8b 0a 85 c9 75 06 c7 02 55 00 00 00 f3 c3 66 0f 1f 44 00 00 <0f> b6 02 48 83 c2 01 48 89 57 18 c3 66 66 66 2e 0f 1f 84 00 00 00
```

using opengl to avoid https://github.com/ValveSoftware/Dota-2-Vulkan/issues/381

[crash_20210724204516_1.dmp.txt](https://github.com/ValveSoftware/Dota-2/files/6872946/crash_20210724204516_1.dmp.txt)

#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): https://gist.github.com/Turmfalke2/c3e1f9ab4e1cf5f6b68ce862c4cf959b
* Have you checked for system updates?: Yes
* Are you using the latest stable video driver available for your system? no
* Have you verified the game files?: no

**Comment by sylware** [comment]
yep, it happens to me randomly when I use a 'going there' ping. We are not alone.
